apps/desktop/src/components/v3/IntegrateUpstreamModal.svelte

```diff
@@ -410,9 +400,7 @@
 				wide
 				style="pop"
 				disabled={isDivergedResolved || !branchStatuses}
-				loading={integratingUpstream === 'loading' ||
-					!branchStatuses ||
-					(forgeListingService && filteredReviews === undefined)}
+				loading={integratingUpstream === 'loading' || !branchStatuses}
```

Comment:
Now `branchStatuses` will never get set before we have finished setting `filteredBranches`. (If there is no forge integration, the function to set `filteredBranches` just returns early)